### ISSUE TITLE
It would be nice if operations that can never be run were set to "errored"

### ISSUE THREAD

**Issue by gasnew** [klavinslab/aquarium-bugs#83]
For example, Make Overnight Suspension in [this plan](http://52.27.43.242:81/launcher?plan_id=24478) can never be moved to pending because its predecessor errored out.

This isn't as simple as erroring all operations in the plan because there may be other goals that can still complete. Thus, I believe this "erroring" method should recurse through all successors to the errored operation.

The main benefit of this enhancement would be to clean up the manager page.

**Comment by wmvoje** [comment]
I agree with this seeing a plan as "error/erroring/errored" would be more clear than seeing "waiting" for operations that will never complete due to a step erroring.
